system/jlib/jfile.cpp

```diff
@@ -152,6 +153,25 @@ StringBuffer &swapPathDrive(StringBuffer &filename,unsigned fromdrvnum,unsigned
 }
 
 
+const char* setStandardPosixPath(StringBuffer &path)
```

Comment:
consider renaming to normalizeToPosixPath or something similar to that, setStandarPosixPath seems to have a different connotation (as far as I'm concerned)